### ISSUE TITLE
Could the crystal heart heals not only players but also horses, villages, cats and dogs?

### ISSUE THREAD

**Issue by Cryghast** [Sollace/Unicopia#184]
They are friends for players, after all.
_Save potion of healing_

**Comment by Sollace** [comment]
I don't know about villagers and horses. I might extend it to include tamed animals though. Will think about it.

**Comment by Sollace** [comment]
I checked on this today. Maybe I did it and forgot, but the heart does already distribute health to all pets.

Namely, the entities included are:
- Horse and Ping (when saddled)
- Cat, Wolf, and Parrot (when tamed)
- Creeper (when tamed by an earth pony)

I'm going to add villagers, so after 1.3.4 it will include:
 - Villager + Wandering Trader

In terms of mods, 
 - anything that has the ability to be saddled (implements Saddleable or extends one of the above)
 - anything that can be samed (extends TameableEntity or one of the above)
 - anything that is a merchant (extends MarchantEntity
are also considered valid targets for distributing health.

Note that health is distributed **equally** between all target entities within range, so the more you have the less effective the heart will be without getting more hostile mobs to balance the supply.

Mobs at full health are not counted towards the demand.
